helm_deploy/probation-teams/templates/_envs.tpl

```diff
@@ -22,6 +22,9 @@ env:
         name: {{ template "app.name" . }}
         key: APPINSIGHTS_INSTRUMENTATIONKEY
 
+  - name: OAUTH_ENDPOINT_URL
```

Comment:
need to remove `JWT_PUBLIC_KEY` from this file too?

Reply:
got rid